data-explorer/kusto/management/alter-merge-mirroring-policy-command.md

```diff
@@ -55,7 +57,7 @@ In the following example, a table called *myTable* is mirrored. The data is part
   partition by (Name: string=Name, Date: datetime= startofday(timestamp))
   dataformat=parquet
   with
-  (IsEnabled=true)
+  (IsEnabled=true, backfill=true)
```

Comment:
I suggest to also add the usage of `EffectiveDateTime` in the example (`EffectiveDateTime=datetime(2025-01-01)"`)